google/cloud/aiplatform/jobs.py

```diff
@@ -2491,10 +2495,11 @@ def update(
                     deployed_model_ids=deployed_model_ids,
                 )
             )
-        self.api_client.update_model_deployment_monitoring_job(
+        response = self.api_client.update_model_deployment_monitoring_job(
```

Comment:
We should add the TODO ticket reference back to to make this method optionally synchronous because this can still be executed asynchronously w.r.t to the update LRO.

Also, preference to capture that this is an lro when making the service call. ie:

```Python
lro = self.api_client.update_model_deployment_monitoring_job(...)
self._gca_resource = lro.result()
```

Reply:
done